Jenkinsfile

```
@@ -16,48 +16,64 @@ pipeline {
     stage('Linting') {
       parallel {
         stage('Code') {
-          steps { sh './bin/test_linting' }
+          steps { echo './bin/test_linting' }
         }
 
         stage('Changelog') {
-          steps { sh './bin/test_changelog' }
+          steps { echo './bin/test_changelog' }
         }
       }
     }
 
     stage('Unit tests') {
       steps {
-        sh './bin/test_unit'
+        echo './bin/test_unit'
       }
       post {
         always {
-          junit 'output/**/*.xml'
-          cobertura autoUpdateHealth: false, autoUpdateStability: false, coberturaReportFile: 'coverage.xml', conditionalCoverageTargets: '50, 0, 50', failUnhealthy: true, failUnstable: false, lineCoverageTargets: '50, 0, 50', maxNumberOfBuilds: 0, methodCoverageTargets: '50, 0, 50', onlyStable: false, sourceEncoding: 'ASCII', zoomCoverageChart: false
-          ccCoverage("coverage.py")
+      echo 'dummy'
+      //junit 'output/**/*.xml'
+          //cobertura autoUpdateHealth: false, autoUpdateStability: false, coberturaReportFile: 'coverage.xml', conditionalCoverageTargets: '50, 0, 50', failUnhealthy: true, failUnstable: false, lineCoverageTargets: '50, 0, 50', maxNumberOfBuilds: 0, methodCoverageTargets: '50, 0, 50', onlyStable: false, sourceEncoding: 'ASCII', zoomCoverageChart: false
+          //ccCoverage("coverage.py")
         }
       }
     }
 
     stage('Integration tests') {
       steps {
-        sh './bin/test_integration'
+        echo './bin/test_integration'
       }
 
       post {
         always {
-          junit 'output/**/*.xml'
+              echo 'dummy'
+          //junit 'output/**/*.xml'
+        }
+      }
+    }
+
+    stage('RHEL7 Integration tests') {
+      steps {
+        sh 'summon -e common ./bin/test_integration_rhel --rhel-version=7'
```

if we are following the pattern of using 'echo'
```suggestion
        echo 'summon -e common ./bin/test_integration_rhel --rhel-version=7'
```